lib/spack/spack/cmd/location.py

```diff
@@ -60,53 +64,68 @@ def setup_parser(subparser):
 
 def location(parser, args):
     if args.module_dir:
-        print(spack.paths.module_path)
+        return print(spack.paths.module_path)
```

Comment:
Returning a print is not clear and I think it's wrong in Python 2 since print is a statement:
```
Python 2.7.17 (default, Feb 27 2021, 15:10:58) 
[GCC 7.5.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> a = print('hello')
  File "<stdin>", line 1
    a = print('hello')
            ^
SyntaxError: invalid syntax
```

Reply:
ugh

Reply:
Tests seem to pass though?

Reply:
I guess it's fine because it uses `from __future__ import print_function`, correct?

Reply:
It works because of that, but I think I never saw returning the value of a `print`. Can't we just:
```python
print(...)
return
```
which is clear on the fact that we don't expect any value to be returned?

Reply:
Done that